.travis.yml

```diff
@@ -1,17 +1,17 @@
 language: python
 python:
-  - "2.7"
-  - "3.4"
+  - 3.6
```

Comment:
Why did you remove python 2.7 and 3.4 from here? It's supposed to build on both python2 and 3

Reply:
Since all of us in the team were using python 3.6, I thought it wasn't a necessary requirement, however, in light of this feedback, let me re-add the python versions